src/app/functions/universal/isMozMail.ts

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const isMozMail = (email: string) => {
+  const emailParsed = email.trim().toLowerCase();
+  return emailParsed.endsWith("@mozilla.com");
```

Comment:
In this case we probably _want_ this restriction, but might be good to point out in a comment perhaps: not all colleagues have an `@mozilla.com` email address (e.g. there's also Pocket and MoFo addresses). None of those need the access granted here, but maybe someone makes assumptions in the future based on the function name.

Reply:
Added a comment in [8c1f676](https://github.com/mozilla/blurts-server/pull/5850/commits/8c1f6764e2088fc88472874714c23d59a9870ed4).